python/py-wmctrl/Portfile

```diff
@@ -0,0 +1,26 @@
+PortSystem          1.0
```

Comment:
please add a "modeline" as the first line in the files, followed by an empty line ([see the guide](https://guide.macports.org/#development.creating-portfile)):
```# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4```

Reply:
Done